contracts/Vault.sol

```diff
@@ -8,22 +8,17 @@ import "./interfaces/IERC4626.sol";
 import "./interfaces/IStrategy.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract Vault is ERC20("Mock cERC20 Strategy", "cERC20", 18), IERC4626, Ownable {
+contract Vault is ERC20("Tracer Vault Token", "TVT", 18), IERC4626, Ownable {
```

Comment:
Do we still need ownable?

Reply:
Yeah the `updatePercentAllocations` function is an `onlyOwner` function